bedrock/firefox/templates/firefox/os/ver/2.5.html

```diff
@@ -0,0 +1,109 @@
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/os/index.html" %}
+
+{% block body_class %}{{ super() }} v2-5{% endblock %}
+
+{% block extrahead %}
+  {% stylesheet 'firefox_os_new' %}
+{% endblock %}
+
+{% block hero_copy %}
+  <div class="hero-content">
+    <h1>{{ _('Experience Firefox OS on your Android device')}}</h1>
+    <p>{{_('Introducing Firefox OS 2.5 Developer Preview, an experimental app that lets you to use Firefox OS on your Android device.')}}</p>
+    <div class="primary-cta-signup signup">
+      <a href="#TODO" class="cta-button" role="button">{{_('Get the Android App')}}</a>
+    </div>
+  </div>
+  <div class="hero-image">
+    {% block phone_image %}
+      {{ high_res_img('firefox/os-new/phone.png') }}
+    {% endblock %}
+  </div>
+{% endblock %}
+
+{% block features_headline %}
+  <h2>{{_('The Web is the platform')}}</h2>
+{% endblock %}
+
+{% block features_copy %}
+  <p>{{_('Firefox OS is the first truly open mobile platform built entirely on Web technologies. If you can build for the Web using HTML and CSS, you can build Firefox OS.')}}
+  <ul class="features">
+    <li class="customization">
+      <h3>{{_('The most customization')}}</h3>
+      <p>{{_('Make Firefox OS your own with apps and add-ons from the Firefox Marketplace, and keep all your favorite items front and centre with Pin the Web.')}}</p>
+    </li>
+    <li class="control">
+      <h3>{{_('The most control')}}</h3>
+      <p>{{_('From Do Not Track to Private Browsing with Tracking Protection, Firefox OS is designed keep you in control of your browsing activity.')}}</p>
+    </li>
+  </ul>
+{% endblock %}
+
+{% block fxox_apps %}
+{% endblock %}
+
+{% block fxox_news %}
+{% endblock %}
+
+{% block fxox_marketplace %}
+{% endblock %}
+
+{% block fxox_hub %}
+  <section class="fxos-hub">
+    <div class="content-wrapper">
+      <div class="hub-container">
+        <h2>{{_('Visit the Participation Hub')}}</h2> 
+        <p>{{_('Want to make Firefox OS even better? Get all the resources, tools and documentation you need from the Participation Hub.')}}</p>
+        <p><a class="more" href="#TODO">{{_('Check it out')}}</a></p>
+      </div>
+      <div class="hub-image">
+        {{ high_res_img('firefox/os-new/hub.png', {'width': '241', 'height': '261'}) }}
+      </div>
+    </div>
+  </section>
+{% endblock %}
+
+{% block fxox_mdn %} 
+  <section class="fxos-mdn">
+    <div class="content-wrapper">
+      <div class="mdn-container">
+        <h2>{{_('Learn more at MDN')}}</h2>
+        <p>{{_('Want to do even more with Firefox OS? Head over to the Mozilla Developer Network to learn more about platform guides, Firefox OS add-ons and more.')}}</p>
+        <p><a class="more" href="#TODO">{{_('Visit MDN')}}</a></p>
+      </div>
+      <div class="mdn-image">
+        {{ high_res_img('firefox/os-new/mdn-logo.png', {'width': '301', 'height': '304'}) }}
```

Comment:
I would usually turn these into backgrounds, but left it the way template 2.0 was originally so it was less work updating. I was thinking these templates need a bit of clean up actually, the blocks are inconsistent, some wrap the whole section, some just the title and content or image. I can file a bug and do some refactoring to these later?